### DISCUSSION TITLE
O componente RaisedButton não funciona

### DISCUSSION THREAD

**Discussion by MarcusMendes81** [Flutterando/forum#102] [Perguntas e Respostas]
Olá pessoal, tudo bem?

Eu sou novo por aqui, estou no processo de aprendizagem ainda. 
Por algum motivo, o componente RaisedButton não está funcionando. Nesse caso, ele não reconhece como um componente. 

`Widget build(BuildContext context) {
    
    return MaterialApp(
        home: Scaffold(
            appBar: AppBar(
              title: Text('Perguntas'),
            ),
            body: Column(
              children: <Widget>[
                Text(perguntas[0]),
                RaisedButton
              ],
            )));
  }`

![image](https://user-images.githubusercontent.com/80123680/207966366-9a112edc-fc00-401c-813c-09f682ecf1b8.png)

  
  O erro que aparece é como se não existisse o componente e ele pede para criar um método, classe ou qualquer outra coisa.
  O que eu faço?
  É necessário importar esse componente? 
  
  Obrigado!

**Comment by eduardoflorence** [comment]
Oi @MarcusMendes81, o RaisedButton foi descontinuado há algum tempo, agora você tem que utilizar o [ElevatedButton](https://api.flutter.dev/flutter/material/ElevatedButton-class.html).

fonte: https://docs.flutter.dev/release/breaking-changes/buttons

**Reply by MarcusMendes81** [reply]
@eduardoflorence obrigado meu caro! vou implementar aqui!
